.github/workflows/updateToLatestRelease.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import json
+
+filepath=input("Enter the file path: ")
+modified_repos = []
+no_releases = []
+path = "https://api.github.com/repos/"
+updateflag = False
+
+with open(filepath) as fhand:
+  line = fhand.readline()
+  cnt = 0
```

Comment:
what's the `cnt` for

Reply:
This can be removed as it is not being used. I was using it while debugging. Will remove that.